packages/pharos-site/src/pages/contributing/documentation.mdx

```diff
@@ -0,0 +1,100 @@
+import PageSection from '../../components/statics/PageSection.tsx';
+
+<PageSection
+  title="Documentation"
+  description="For all Pharos components and elements, contributors create useful, thorough documentation to provide an understanding of its proper usage and context."
+  isHeader
+></PageSection>
+<PageSection
+  title="Importance"
+  description="Documentation is essential to upholding quality and cohesion. 
+Articulate and thorough documentation allows for users and contributors to acquire knowledge more efficiently, motivating further contribution and information sharing. 
+Referring to documentation helps to reduce redundancy in components and features while ensuring all implemented features are being used appropriately and to their 
+fullest potential."
+></PageSection>
+<PageSection
+  title="When documentation is needed"
+  description="When a new component or feature is added, supporting documentation should be written to communicate and provide guidelines for its proper usage. 
+Additionally, if changes are made to existing components that affect their usage or functionality, the documentation should reflect these changes."
+></PageSection>
+<PageSection title="Writing documentation">
+  <PageSection title="Getting started" subSectionLevel={1}>
+    <p>
+      To get started, use our
+      <PharosLink href="https://docs.google.com/document/d/1GDFM4iQsta8MjErF7hEYkFaNMoQNKzEzpM-RWG8JcwM/edit">
+        {' '}
+        documentation template{' '}
```

Comment:
ill get rid of these

Reply:
Think you want those for the spaces between the link and text